dev_setup.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
```

Comment:
In general I'd recommend sticking to #!/bin/sh and avoiding any bashisms (unless there's some specific reason to need bash, which seems unlikely)

... otherwise "install bash" can end up being a prereq for running the setup script and we don't want that.

Reply:
Thanks for pointing this out! I think I got all of the bashisms out

Reply:
I found one more, but I gave it its own comment so I'll mark this one done

Reply:
Thanks!